protocol/protocolwrapper/protocol_filter_wrapper_test.go

```diff
@@ -30,17 +30,42 @@ import (
 	"github.com/apache/dubbo-go/common"
 	"github.com/apache/dubbo-go/common/constant"
 	"github.com/apache/dubbo-go/common/extension"
-	"github.com/apache/dubbo-go/filter/impl"
+	"github.com/apache/dubbo-go/filter"
+	//"github.com/apache/dubbo-go/filter/impl"
 	"github.com/apache/dubbo-go/protocol"
 )
 
+//To avoid import cycle
```

Comment:
import cycle should not be exist.
And, this mock is meaningless, you can modify `impl.ECHO` to `"echo"`.